File: CrimeSearch/ClientApp/src/models/searchParameter.ts
export class SearchParameter {
  public fieldName: string;
  public searchOperator: string;
  public searchValue: string;
  public fieldType: string;
  public displayName: string;

  public constructor(fieldName: string, operator: string, searchValue: string, fieldType: string, displayName: string) {
    this.fieldName = fieldName;
    this.searchOperator = operator;
    this.searchValue = searchValue;
    this.fieldType = fieldType;
    this.displayName = displayName;
  }
}

File: CrimeSearch/Models/SearchParameter.cs
namespace CrimeSearch.Models
{
    public class SearchParameter
    {
        public string FieldName { get; set; }
        public string SearchOperator { get; set; }
        public string SearchValue { get; set; }
        public string FieldType { get; set; }
        public string ConjunctiveOperator { get; set; }
    }
}
